spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -175,6 +175,14 @@ for _, strategy in helpers.each_strategy() do
           end
         end, 5)
       end)
+
+      it("local cached config file has correct permission", function()
+        local handle = io.popen("stat servroot2/config.cache.json.gz")
```

Comment:
`stat` isn't POSIX, it's part of the GNU coreutiils.  do we run on any BSD system?  (i guess Darwin does install GNU tools now)

Reply:
> do we run on any BSD system?

Some devs run the test suite on macOS. `stat` flavors are always a portability annoyance. A simple `ls -l` should be enough for the purposes of this test.

Reply:
Actually on macOS 10.15, `stat` is already pre-installed, so it should not pose any challenges for running on macOS.

```
~ datong$ which stat
/usr/bin/stat
```

Reply:
Changed to `ls -l`.